helm/prometheus-rules/templates/kaas/phoenix/alerting-rules/aws.job.rules.yml

```diff
@@ -0,0 +1,29 @@
+## TODO Remove with vintage
+# This rule applies to vintage aws management clusters
+{{- if eq .Values.managementCluster.provider.flavor "vintage" }}
```

Comment:
Don't we need `{{- if eq .Values.managementCluster.provider.kind "aws" }}` as before? Or we're fine that `cronjob="route53-manager"` won't find anything on other MCs?

Reply:
We're fine with it, this is just to make sure we do not deploy the rules on capa and other mcs because we do not need it. But it's UP to you. Let me know what you prefer :)

Reply:
Oh, i misunderstood, we're fine with it because aws is the only remaining vintage provider (because KVM alerts are coming from a custom branch and azure is gone)